### PR TITLE
feat: stylish-haskell support

### PR DESCRIPTION
**Pull request by bbarks15**

### CODE REVIEW

**Comment by stevearc** [comment]
You linked to the fourmolu formatter, is that intended?

**Comment by brandonbarkerRAC** [comment]
No that is not intended 😂

Will get that fixed

**Comment by bbarks15** [comment]
Was not intended lol

Fixed

File: lua/conform/formatters/stylish-haskell.lua [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://hackage.haskell.org/package/stylish-haskell",
+    description = "A simple Haskell code prettifier",
+  },
+  command = "stylish-haskell",
+  args = { "$FILENAME" },
```
Comment (stevearc) [inline lua/conform/formatters/stylish-haskell.lua:8]: From the docs, it looks like `stylish-haskell` supports formatting via stdin & stdout. Do you need this arg?

**Review by stevearc — CHANGES_REQUESTED** [review]

**Comment by stevearc** [comment]
Implemented in #435

**Pull request closed without merging**